jetty-quic/quic-server/src/main/java/org/eclipse/jetty/quic/server/QuicServerConnector.java

```diff
@@ -163,19 +164,32 @@ protected void doStart() throws Exception
             throw new IllegalStateException("Invalid KeyStore: no aliases");
         String alias = sslContextFactory.getCertAlias();
         if (alias == null)
-            alias = aliases.stream().findFirst().orElse("mykey");
-        char[] keyStorePassword = sslContextFactory.getKeyStorePassword().toCharArray();
+            alias = aliases.stream().findFirst().orElseThrow();
         String keyManagerPassword = sslContextFactory.getKeyManagerPassword();
-        SSLKeyPair keyPair = new SSLKeyPair(
-            sslContextFactory.getKeyStoreResource().getFile(),
-            sslContextFactory.getKeyStoreType(),
-            keyStorePassword,
-            alias,
-            keyManagerPassword == null ? keyStorePassword : keyManagerPassword.toCharArray()
-        );
-        File[] pemFiles = keyPair.export(new File(System.getProperty("java.io.tmpdir")));
-        privateKeyFile = pemFiles[0];
-        certificateChainFile = pemFiles[1];
+        char[] password = keyManagerPassword == null ? sslContextFactory.getKeyStorePassword().toCharArray() : keyManagerPassword.toCharArray();
+        KeyStore keyStore = sslContextFactory.getKeyStore();
+        Path certificateWorkPath = findCertificateWorkPath();
+        Path[] keyPair = PemExporter.exportKeyPair(keyStore, alias, password, certificateWorkPath);
+        privateKeyPemPath = keyPair[0];
+        certificateChainPemPath = keyPair[1];
+        KeyStore trustStore = sslContextFactory.getTrustStore();
+        if (trustStore != null)
+            trustedCertificatesPemPath = PemExporter.exportTrustStore(trustStore, certificateWorkPath);
+    }
+
+    private Path findCertificateWorkPath()
```

Comment:
Rename for consistency to `findPemWorkDirectory()` like we have in `QuicConfiguration`.

Reply:
Done.